tests/ctst/world/Zenko.ts

```diff
@@ -99,6 +107,322 @@ export default class Zenko extends World {
         return decision;
     }
 
+    /**
+     * This function will dynamically prepare credentials based on the type of
+     * entity provided to let the test run the AWS CLI command using this particular
+     * type of entity.
+     * @param {ScenarioCallerType} type - type of entity, can be 'account', 'storage manager',
+     * 'storage account owner', 'data consumer' or 'iam user'
+     * @returns {undefined}
+     */
+    async prepareForType(type: string): Promise<void> {
+        const savedParameters = JSON.parse(JSON.stringify(this.cliOptions));
+        this.resetGlobalType();
+        switch (type) {
+            case EntityType.ACCOUNT:
+                await this.prepareRootUser();
+                this.saved.type = EntityType.IAM_USER;
+                break;
+            case EntityType.IAM_USER:
+                await this.prepareIamUser();
+                this.saved.type = EntityType.ACCOUNT;
+                break;
+            case EntityType.STORAGE_MANAGER:
+                await this.prepareARWWI(
+                    this.parameters.StorageManagerUsername || 'storage_manager',
+                    this.parameters.KeycloakTestPassword || '123',
```

Comment:
I think I already reviewed this file but in another PR, so putting the comments here again:

We can store the default password in a const

Reply:
yes branch wasn't rebased